lasagne/init.py

```diff
@@ -1,5 +1,11 @@
 """
-Functions to create initializers for parameter variables
+Functions to create initializers for parameter variables.
+
+Usage
+-------
+>>> from lasagne.layers import DenseLayer
+>>> from lasagne.init import Constant, Glorot
+>>> l1 = DenseLayer((100,20), num_units=50, W=Glorot(), b=Constant(0.0))
```

Comment:
Now that you removed the default initializer, you cannot use `Glorot()` any more, but have to use one of the two shortcuts. Bonus points if you merge this with the previous commit :) (Just because it's a good opportunity to practice this. `git add`, `git commit --amend` and `git push --force` are your friends if you want to try it.)